prestoadmin/standalone/config.py

```diff
@@ -116,6 +118,7 @@ def validate(conf):
     except KeyError:
         pass
     else:
+        workers = [h for host in workers for h in expand_host(host)]
```

Comment:
what if I don't want name expansion like that?
For example what if I have host names which are exactly like `cdh5-1` and `cdh5-2`?